hrjobcontract/scss/civihr/partials/_forms.scss

```diff
@@ -32,6 +35,7 @@ input[type="checkbox"],
   }
 
   .has-feedback {
+    /* stylelint-disable selector-max-compound-selectors */
```

Comment:
shouldn't this be `disable-line`

Reply:
No inside this block, there are multiple instances of more than 3 compound selectors, so disabling all at once.